modules/launcher/src/main/resources/broker.yaml

```diff
@@ -60,6 +60,19 @@ ballerina.broker.auth:
     # Optional properties
     properties:
      loginModule: io.ballerina.messaging.broker.auth.authentication.jaas.UserStoreLoginModule
+ # Broker authorization related configurations.
+ authorization:
+  # Enable the authorization
+  enabled: true
+  authProvider:
+   # AuthProvider implemetation
+   className: io.ballerina.messaging.broker.auth.authorization.provider.UserStoreAuthProvider
```

Comment:
This should be a property in the Authorizor class.

Reply:
Fixed in ebaf38d